entities/book.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require_relative '../dependencies'
+
+# books to read
+class Book
+  include Validate
```

Comment:
Here you require validate module
https://github.com/VKuzmich/Library_RG/pull/1/files#diff-c9e85b3f1fe7f7cbc1a0bc47a6bb3ed4R3
but in this file you don't. why?

Reply:
fixed